src/services/encryption/password.ts

```diff
@@ -29,8 +29,11 @@ function changePassword(currentPassword, newPassword) {
         optionService.setOption('passwordDerivedKeySalt', utils.randomSecureToken(32));
 
         const newPasswordVerificationKey = utils.toBase64(myScryptService.getVerificationHash(newPassword));
-
-        passwordEncryptionService.setDataKey(newPassword, decryptedDataKey);
+        
+        if (decryptedDataKey) {
+            // FIXME: what should happen if the decrypted data key is null?
```

Comment:
Nit: Comments fade into the background, using TODO would change the color of this for those of us who use [Better Comments](https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments). Not that we need to conform a tool, but conforming to a standard could be useful for those of us who do use tools for comments!

Alternatively, new tags/styles can be added, but should perhaps be added to the workspace settings so all can benefit from them.

```suggestion
            // TODO: what should happen if the decrypted data key is null?
```

Reply:
@j-tirelli , done.

Reply:
I can't resolve comments, but happy to make them :)

Reply:
@j-tirelli , I've just gave you dev rights, let me know if you can resolve them now. :)